docs/THIRDPARTY

```diff
@@ -116,6 +116,11 @@ Files: static/third/marked/*
 Copyright: 2011-2013, Christopher Jeffrey
 License: Expat
 
+Files: static/assets/icons/ellipsis-v-solid.svg
+Copyright: Fonticons, Inc.
+License: Creative Commons Attribution 4.0 International
```

Comment:
This file has a standard machine-readable [format](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/), and this license should be [specified](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-short-name) as `CC-BY-4.0`.

Reply:
ohh... I will make those changes